_includes/releases/v21.2/v21.2.14.md

```diff
@@ -0,0 +1,87 @@
+## v21.2.14
+
+Release Date: August 1, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v21-2-14-security-updates">Security updates</h3>
+
+- Multi-region related built-in functions now have access control checks. [#83987][#83987]
```

Comment:
Could link to functions-and-operators.html#multi-region-functions? I assume this is what the note is referring to..